File: 프로그래머스/0/181898. 가까운 1 찾기/가까운 1 찾기.py
def solution(arr, idx):
    # method1
    r=-1
    for i in range(idx,len(arr)):
        if arr[i]==1:
            r=i
            break
    # method2
    # try:
    #     r=arr.index(1,idx)
    # except:
    #     r=-1    
    return r

File: 프로그래머스/0/181898. 가까운 1 찾기/가까운 1 찾기.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

// arr_len은 배열 arr의 길이입니다.
int solution(int arr[], size_t arr_len, int idx) {
    for(int i=idx;i<arr_len;i++)
        if(arr[i]==1)
            return i;
    return -1;
}